client-app/src/desktop/common/Wrapper.js

```diff
@@ -20,10 +54,70 @@ class Wrapper extends Component {
                     marginBottom: 10,
                     omit: !description
                 }),
-                children
+                children,
+                this.renderSourceCodePanel()
             ],
             ...rest
         });
     }
+    
+    renderSourceCodePanel() {
+        const {dockContainerModel} = this;
+        const {links, link} = this.props;
+        
+        if (links || link) {
+            dockContainerModel.addView({
+                id: XH.genId(),
+                icon: Icon.code(),
+                title: code('Source Code'),
+                allowDialog: false,
```

Comment:
I think these docked views could be initialised with `collapsed: true`

Reply:
👍